Mathlib/Algebra/Lie/Submodule.lean

```diff
@@ -729,11 +729,13 @@ def lieSpan : LieSubmodule R L M :=
 variable {R L s}
 
 theorem mem_lieSpan {x : M} : x ∈ lieSpan R L s ↔ ∀ N : LieSubmodule R L M, s ⊆ N → x ∈ N := by
-  change x ∈ (lieSpan R L s : Set M) ↔ _; erw [sInf_coe]; exact mem_iInter₂
+  change x ∈ (lieSpan R L s : Set M) ↔ _
```

Comment:
Feel free to tackle this `change` as well

Reply:
I'll leave this for someone else. :-)